actions/package/publish/action.yml

```diff
@@ -0,0 +1,46 @@
+name: Package Grafana Backend
+inputs:
+  grafana_ref:
+    description: 'The commit to use when checking out Grafana'
+    required: true
+    default: 'main'
+  enterprise:
+    description: 'Whether or not to use Grafana Enterprise. Supplying this argument means that you have a GitHub token that is capable of cloning Grafana Enterprise'
+    required: false
+  enterprise_ref:
```

Comment:
Would it perhaps be more consistent to name that one `grafana_enterprise_ref`?

Reply:
not that it matters very much but I'd prefer not I think. The word "enterprise" to me isn't ambiguous.

I don't feel strongly about it either way though so if you do then I am happy to change it!

Reply:
On second thought, my main issue is actually with `grafana_ref` 😅 Perhaps something like `ref` for the normal Grafana reference and `enterprise_ref` for the Grafana Enterprise reference?

Reply:
I agree with you, however I think after our very thorough conversation in slack I will leave it. My reason being this:

This is ugly:

```
--grafana-ref=1234
--enterprise=true
--enterprise-ref=asdf
```

and it makes more sense to do this, since this project is literally for grafana. The "grafana" is kind of implied.

```
--ref=1234
--enterprise=true
--enterprise-ref=1234
```

BUT... if we have both an `--enterprise` and `--grafana` flag, then this makes the most sense maybe?

```
--grafana=true
--grafana-ref=1234
--enterprise=true
--enterprise-ref=asdf
```